eZ/Bundle/EzPublishCoreBundle/Fragment/FragmentListenerFactory.php

```diff
@@ -23,12 +23,14 @@ class FragmentListenerFactory
 
     public function buildFragmentListener(UriSigner $uriSigner, $fragmentPath, $fragmentListenerClass)
     {
-        // Ensure that current pathinfo ends with configured fragment path.
-        // If so, consider it as the fragment path.
-        // This ensures to have URI siteaccess compatible fragment paths.
-        $pathInfo = $this->getCurrentRequest()->getPathInfo();
-        if (substr($pathInfo, -strlen($fragmentPath)) === $fragmentPath) {
-            $fragmentPath = $pathInfo;
+        if ($request = $this->getCurrentRequest()) {
```

Comment:
I don't think we allow assigment in if statements in the CS.
You should move this up dans do `if ($request)`